src/pipeline/rackhd/ova/FunctionTest.groovy

```diff
@@ -29,12 +45,13 @@ def runTest(String stack_type, String test_name, ArrayList<String> used_resource
                 String library_dir = "$WORKSPACE/on-build-config"
                 shareMethod.checkoutOnBuildConfig(library_dir)
                 String ova_path = ""
+                println ova_url
                 if (ova_url != ""){
                     ova_path = ova_url
                 } else {
-                    ova_path = shareMethod.unstashFile(ova_dict, "$WORKSPACE")
+                    ova_path = shareMethod.unstashFile(ova_stash_dict, "$WORKSPACE")
                 }
-
+                println ova_path
                 String rackhd_dir = "$WORKSPACE/RACKHD"
                 shareMethod.checkout("https://github.com/changev/RackHD.git", "debug/ssh", rackhd_dir)
```

Comment:
Your PR from debug/ssh is already merged.
Please use RackHD/RackHD and master branch